shallow_backup/prompts.py

```diff
@@ -14,14 +16,20 @@ def path_update_prompt(config):
 	"""
 	current_path = config["backup_path"]
 	print_path_blue("Current shallow-backup path:", current_path)
-	if prompt_yes_no("Would you like to update this?", Fore.GREEN):
-		print_green_bold("Enter relative or absolute path:")
-		abs_path = expand_to_abs_path(input())
-		print_path_blue("\nUpdating shallow-backup path to:", abs_path)
-		mkdir_warn_overwrite(abs_path)
-		move_git_repo(current_path, abs_path)
-		config["backup_path"] = abs_path
-		write_config(config)
+	if prompt_yes_no("Would you like to update this?", Fore.GREEN, invert=True):
+		while True:
+			print_green_bold("Enter relative or absolute path:")
+			abs_path = expand_to_abs_path(input())
+
+			if new_dir_is_valid(abs_path):
```

Comment:
I'm not sure this logic is right.

```suggestion
			if not new_dir_is_valid(abs_path):
			    # Print error saying dir is invalid, ask for reprompt
```